src/test/java/com/google/javascript/clutz/types_externs_with_platform_usage.ts

```diff
@@ -1,4 +1,5 @@
 import {elementMaybe, Error} from 'goog:typesWithExterns';
 
-var el: Element = elementMaybe();
+//!! TS Element disagrees with Closure's Element because of missing msContentZoomFactor field.
+var el: Element = <any> (elementMaybe());
```

Comment:
As discussed offline, making the DOM incompatible seems like a very bad outcome. We should reconsider how hard it'd be to make `lib.d.ts` and Closure's externs match.